dfm/description/description_pattern_lists.py

```diff
@@ -0,0 +1,436 @@
+occupation_pattern_list = set([
```

Comment:
You typically don't want data in scripts. I suggest writing these to .json files and read them in.